packages/pwa/app/utils/locale.js

```diff
@@ -7,16 +7,24 @@
 
 import {SUPPORTED_LOCALES, DEFAULT_LOCALE} from '../constants'
 
+const supportedLocales = SUPPORTED_LOCALES.map((locale) => locale.id)
+
+/**
+ * @returns {string[]} short codes of all the app's supported locales
+ */
+export const getSupportedLocales = () => supportedLocales
```

Comment:
nit: Since we're doing the configuration refactor, this is a moving target, but I'd change the `getSupportedLocalesIds` and `supportedLocalesIds` to differentiate better from the raw constant `SUPPORTED_LOCALES`.

Reply:
Ok, good point. I'll make the change.

Reply:
I've replaced the exported function `getSupportedLocales` with `getSupportedLocalesIds` as you suggested.

However, I didn't change the name of local variables (like `supportedLocales`). I'll leave it up to the files/components how they name their local variables.